src/doom/r_segs.c

```diff
@@ -197,6 +197,24 @@ R_RenderMaskedSegRange
 #define HEIGHTBITS		12
 #define HEIGHTUNIT		(1<<HEIGHTBITS)
 
+// Doom 1.2 uses 16.16 fixed point numbers here
+int heightbits = HEIGHTBITS;
+fixed_t heightunit = HEIGHTUNIT;
+
+void R_SetHeightFracBits(void)
+{
+    // Moire error emulation
+    if (gameversion <= exe_doom_1_2)
+    {
+        heightbits = 16;
```

Comment:
Can you make these into constants? eg.

    #define HEIGHTBITS           12
    #define HEIGHTBITS_DOOM12    16